tests/test_cli.py

```diff
@@ -1116,6 +1127,47 @@ def test_run_with_defaults_and_env_from_config_file(self, skipper_runner_run_moc
                                                         interactive=False, name=None, net='host', volumes=None,
                                                         workdir=None, workspace=None, use_cache=False)
 
+    @mock.patch('__builtin__.open', mock.MagicMock(create=True))
+    @mock.patch('os.path.exists',
+                mock.MagicMock(autospec=True, return_value=True))
+    @mock.patch('yaml.safe_load', mock.MagicMock(
+        autospec=True,
+        return_value=SKIPPER_CONF_WITH_ENV_FILE
+    ))
+    @mock.patch('skipper.utils.load_yaml_file', mock.MagicMock(
+        autospec=True,
+        return_value=CONFIG_ENV_EVALUATION
```

Comment:
I think you should test:
1. validate new envs from the file.
2. validate envs from the file with overrides in the command line (command line should take precedence)
3. Empty env file
4. malformed env file